hooked/views.py

```diff
@@ -72,9 +73,12 @@ def check_webhook_signature(self, request):
             return
 
         self.log_failure(
-            'Mismatch between computed and claimed signature of incoming '
-            'events. I computed {}, but the HTTP header said I should '
-            'expect to find {}'.format(computed_signature, claimed_signature))
+            message='Mismatch between computed and claimed signature of incoming events',
+            extra={
+                'expected_signature': computed_signature,
+                'actual_signature': claimed_signature
+            }
+        )
```

Comment:
That's a good point actually... you can't do much other than observe the non-matchingness for yourself. I guess if something completely wacky happened one of expected or computer may wind up as `None` or something?